ui/app/mirrors/[mirrorId]/page.tsx

```diff
@@ -18,13 +18,20 @@ type EditMirrorProps = {
   params: { mirrorId: string };
 };
 
-function getMirrorStatusUrl(mirrorId: string) {
+const obtainMirrorName = (jobName: string) => {
+  if(jobName.startsWith("clone_")){
+    return jobName.split("_")[1];
```

Comment:
mirrors can have underscores in them, this fails then

Reply:
jobName.slice("clone_".length)